### ISSUE TITLE
[Feature Request] Map ocarina notes to d-pad when ocarina is out

### ISSUE THREAD

**Issue by Bordraw** [Mr-Wiseguy/Zelda64Recomp#275]
I just downloaded 1.1.0 and the new camera works great, but I was depending on the right stick to use for my ocarina inputs. Its hard to find a mapping on my standard xbox pattern controller that is good for both using items and playing the ocarina when the right stick is dedicated to the camera.

A good solution to this would be to add an option to contextually use the d-pad for ocarina playing. This could be exposed as a toggleable option in the general settings page.

**Comment by Mr-Wiseguy** [comment]
Ocarina inputs on the right stick will still work even with dual analog active. Dual analog mode automatically disables inputs on the right stick outside of the ocarina, so there's no need to change your controller mapping.

**Comment by Bordraw** [comment]
Oh, excellent! Maybe that should be included in the description of the setting?

I still think having this binding option would make sense, but its lower priority if it isn't blocked on the right stick camera

**Comment by Bordraw** [comment]
I just experimented a little and realised that the key item buttons can be double mapped with the c buttons to do the thing I'm trying to do anyways, so this feature is unnecessary. This can be closed now.
